composer.json

```diff
@@ -19,7 +19,7 @@
     "require": {
         "php": ">=5.5",
         "container-interop/container-interop": "^1.1",
-        "zendframework/zend-expressive": "^1.0@rc",
+        "zendframework/zend-expressive": "~1.0.0-dev@dev",
```

Comment:
It will essentially grab dev-master always. I think instead I'm going to tag the zend-expressive repo, and then update this one to the latest tag.

I understand why everyone wanted to separate out the implementations, but while we're not stable, this is a nightmare to maintain the dependencies!